src/main/java/duke/commands/UnmarkCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke.commands;
+
+import duke.DukeException;
+import duke.common.Messages;
+import duke.storage.StorageFile;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.TextUi;
+
+/**
+ * Unmark a task in the task list.
+ */
+public class UnmarkCommand extends Command {
+
+    public static final String COMMAND_WORD = "unmark";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Unmark a task (not done).\n"
+            + "\tEx.: " + COMMAND_WORD + " <number>";
+
+    private final int targetIndex;
+
+    /**
+     * Constructor for UnmarkCommand.
+     *
+     * @param targetIndex index of the task to be unmarked
+     */
+    public UnmarkCommand(int targetIndex) {
+        this.targetIndex = targetIndex;
+    }
+
+    @Override
+    public CommandResult execute(TaskList taskList, TextUi textUi, StorageFile storage) {
```

Comment:
Should we perhaps add JavaDocs here as well, since there are multiple different parts being involved here?